src/SprykerEco/Yves/Computop/Handler/ExpressCheckout/ComputopPayPalExpressCompleteHandler.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Handler\ExpressCheckout;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerEco\Client\Computop\ComputopClientInterface;
+use SprykerEco\Yves\Computop\Dependency\Client\ComputopToComputopApiClientInterface;
+
+class ComputopPayPalExpressCompleteHandler implements ComputopPayPalExpressCompleteHandlerInterface
```

Comment:
Handler is to general name for a business model, we also have convention for this https://spryker.atlassian.net/wiki/spaces/CORE/pages/478052552/Application+Namespace+Conventions#ApplicationNamespaceConventions-BusinessModel

Reply:
Already discussed